scripts/gen-keybinding-img

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/env python
+
+#######################################
+#  Qtile keybindings image generator  #
+#######################################
+
+import getopt
+import os
+import sys
+import cairocffi as cairo
+from cairocffi import ImageSurface
+
+this_dir = os.path.dirname(__file__)
+base_dir = os.path.abspath(os.path.join(this_dir, ".."))
+sys.path.insert(0, base_dir)
+
+BUTTON_NAME_Y = 65
+BUTTON_NAME_X = 10
+
+COMMAND_Y = 20
+COMMAND_X = 10
+
+LEGEND = ['modifiers', 'layout', 'group', 'window', 'other']
+
+CUSTOM_KEYS = {
+    'Backspace': 2,
+    'Tab': 1.5,
+    '\\': 1.5,
+    'Return': 2.45,
+    'shift': 2,
+    'space': 5,
+    }
+
+
+class Button:
+    def __init__(self, key, x, y, width, height):
+        self.key = key
+        self.x = x
+        self.y = y
+        self.width = width
+        self.height = height
+
+
+class Pos:
+    WIDTH = 78
+    HEIGHT = 70
+    GAP = 5
+
+    def __init__(self, x, y):
+        self.x = x
+        self.row_x = x
+        self.y = y
+        self.custom_width = {}
+        for i, val in CUSTOM_KEYS.items():
+            self.custom_width[i] = val * self.WIDTH
+
+    def get_pos(self, name):
+        if name in self.custom_width:
+            width = self.custom_width[name]
+        else:
+            width = self.WIDTH
+
+        info = Button(name, self.x, self.y, width, self.HEIGHT)
+
+        self.x = self.x + self.GAP + width
+
+        return info
+
+    def skip_x(self, times=1):
+        self.x = self.x + self.GAP + times * self.WIDTH
+
+    def next_row(self):
+        self.x = self.row_x
+        self.y = self.y + self.GAP + self.HEIGHT
+
+
+class kbPngFactory:
```

Comment:
This class name isn't very Pythonic, what do you think of something like this instead?

```suggestion
class KeyboardPNGFactory:
```

Reply:
will refactor